packages/patternfly-4/react-table/src/components/Table/Body.tsx

```diff
@@ -125,11 +127,14 @@ class ContextBody extends React.Component<TableBodyProps, {}> {
 }
 
 export const TableBody = ({
-  onRow = (...args: any) => Object,
   className = '' as string,
   children = null as React.ReactNode,
   rowKey = 'id' as string,
-  onRowClick = (event: React.MouseEvent, row: IRow, rowProps: IExtraRowData, computedData: IComputedData) => undefined as OnRowClick,
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  onRow = (...args: any) => Object,
```

Comment:
`onRow` and `onRowClick` appear to be used?

Reply:
Both onRow and onRowClick were throwing TS errors - it looks like the unused variable errors were for the arguments to both functions (...args for onRow, and event/row/rowProps/computedData for onRowClick)

Reply:
Can we track down how `onRow` is used and give it a proper type?

Also, for the unused variables, just put underscores before the variable names. Like `_event: React.MouseEvent`. Then you can remove the eslint-disable.